src/modlunit/units.cpp

```diff
@@ -109,28 +113,25 @@ static int Getc(FILE* inp) {
 #define UNIT_STK_SIZE 20
 static struct unit unit_stack[UNIT_STK_SIZE], *usp{nullptr};
 
-static char* neuronhome() {
+static std::string hoc_dos2unixpath(std::string d) {
+    std::replace(d.begin(), d.end(), '\\', '/');
```

Comment:
Can we use `std::filesystem::path` instead, at least internally? I'm not a huge fan of string manipulation for paths, especially if there's a native way of handling them.

Reply:
it seems to lack a lot of method for converting path between platform...